File: src/main/webapp/js/Integration/Address.js
function Address(use, line, city, state, postalCode) {
	this.use = use;
	this.line = line;
	this.city = city;
	this.state = state;
	this.postalCode = postalCode;
};

Address.prototype.prettyPrint = function() {
	return "Use: " + this.use
	+ " Street: " + this.line
	+ " City: " + this.city
	+ " State: " + this.state
	+ " Postal Code: " + this.postalCode;
};

Address.prototype.jsonPrint = function() {
	return { use: this.use,
				line: this.line,
				city: this.city,
				state: this.state,
				postalCode: this.postalCode
			};
};

module.exports = Address;

File: src/main/java/Entities/Address.java
package Entities;

import org.json.*;

public class Address {

	private static final String POSTAL_CODE = "postalCode";
	private static final String STATE = "state";
	private static final String CITY = "city";
	private static final String LINE = "line";
	private static final String USE = "use";
	private JSONObject address;

	public Address(String use, String line, String city, String state, String postalCode) {
		address = new JSONObject();
		address.put(USE, use);
		JSONArray lineArray = new JSONArray();
    	lineArray.put(line);
		address.put(LINE, lineArray);
		address.put(CITY, city);
		address.put(STATE, state);
		address.put(POSTAL_CODE, postalCode);
	}
	
	public JSONObject getJSONObject() {
		return address;
	}
}